cmd/algorand-indexer/daemon.go

```diff
@@ -154,17 +154,17 @@ type blockImporterHandler struct {
 	cache map[uint64]bool
 }
 
-func (bih *blockImporterHandler) HandleBlock(block *types.EncodedBlockCert) {
+func (bih *blockImporterHandler) HandleBlock(block *rpcs.EncodedBlockCert) {
 	start := time.Now()
 	_, err := bih.imp.ImportDecodedBlock(block)
-	maybeFail(err, "ImportDecodedBlock %d", block.Block.Round)
+	maybeFail(err, "ImportDecodedBlock %d", block.Block.Round())
 	metrics.BlockUploadTimeSeconds.Observe(time.Since(start).Seconds())
 	startRound, err := bih.db.GetNextRoundToAccount()
 	maybeFail(err, "failed to get next round to account")
 	// During normal operation StartRound and MaxRound will be the same round.
 	filter := idb.UpdateFilter{
 		StartRound: startRound,
-		MaxRound:   uint64(block.Block.Round),
+		MaxRound:   uint64(block.Block.Round()),
```

Comment:
it's not incorrect, but please aware that the round field still presents on `block.Block.BlockHeader.Round`